handbook/engineering/search/index.md

```diff
@@ -12,22 +12,24 @@ Sourcegraph search is:
 - **Easy to use:** Users can quickly understand how to find what they are looking for and what options are available for searching. The search syntax is clear and intuitive.
 - **Shareable:** Sourcegraph searches are easy to share, and provide team and organization wide value. This in turn creates network effects that compound the value Sourcegraph provides with scale.
 
+## Current focus
+
+Our current focus is documented in the [tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+
+
 ## Goals
 
-_Updated 2020-09-05_
+_Updated 2020-09-11_
 
-## [WIP] Medium term (3-6 months)
+### Scalable
 
-### Scale indexed search to 500k repositories
+- **Scale indexed search to 500k repositories**
    - **Problem:** We have customers who need Sourcegraph to scale to 500k repositories. Some parts of Sourcegraph don't work well at that scale.
    - **Outcome:** Sourcegraph can search 500k repositories in less than 300ms. This is on par with [grep.app](https://grep.app).
    - **Plan:** Incrementally add repositories to Sourcegraph.com until searches get slow or start breaking. Fix those things. Then continue adding repositories.
-   - **Owners:** TBD
+   - **Owners:** Keegan/Stefan
    - **Status:** Sourcegraph.com currently has about 100k repositories.
-
-## Short term (1-3 months)
-
-Our current focus is documented in the [tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+   - **Estimated completion**: TODO
```

Comment:
@keegancsmith @stefanhengl could you add an (even rough) estimated completion milestone for this work (eg. `3.22`)? Thanks!

Reply:
I don't know when we will reach the goal. Maybe it is better to dedicate time up front (3 months?). That is to avoid working on it forever, but still give a timeframe where we can be more ambitious. Then obviously as we learn more / priorities change we can adjust. WDYT?

Reply:
Yeah, I understand the unknowns here -- just shooting for a rough ballpark mostly to communicate team availability / bandwidth for the next few milestones effectively. Putting down 3 months as an initial estimate then refining it as we go sounds good to me 👍